todoman/ui.py

```diff
@@ -258,41 +265,67 @@ def detailed(self, todo):
             rv = "{}\n\n{}".format(rv, todo.description)
         return rv
 
-    def format_date(self, date):
+    def _format_date(self, date):
         """
         Returns date in the following format:
         * if date == today or tomorrow: "Today" or "Tomorrow"
         * else: return a string representing that date
         * if no date is supplied, it returns empty_date
 
-        :param datetime.datetime date: a datetime object
+        :param datetime.date date: a date object
         """
         if date:
-            assert isinstance(date, datetime)
-            if date.date() in self.special_dates:
-                rv = self.special_dates[date.date()]
+            assert type(date) is datetime.date
```

Comment:
We don't really need this assertion any more due to how we call this function (we've also made it sort of non-public now).